gcloud/bigtable/test_cluster.py

```diff
@@ -53,6 +53,30 @@ def test_constructor_non_default(self):
         self.assertEqual(cluster.serve_nodes, serve_nodes)
         self.assertTrue(cluster._client is client)
 
+    def test_copy(self):
+        project = 'PROJECT'
+        zone = 'zone'
+        cluster_id = 'cluster-id'
+        display_name = 'display_name'
+        serve_nodes = 8
+
+        client = _Client(project)
+        cluster = self._makeOne(zone, cluster_id, client,
+                                display_name=display_name,
+                                serve_nodes=serve_nodes)
+        new_cluster = cluster.copy()
+
+        # Make sure the client got copied to a new instance.
+        self.assertFalse(new_cluster._client is client)
+        self.assertEqual(new_cluster._client.__dict__,
+                         client.__dict__)
+
+        # Just replace the client on the new_cluster so we can
```

Comment:
Because `Cluster.__eq__` only checks `==` for the clients and I never implemented `_Client.__eq__` for the test stub.

It seems I should just implement `__eq__`, no?

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/f575283235bb87f6bbc3a989c4b474d9eacbae1b/gcloud/bigtable/cluster.py#L180-L182